relevanceai/datasets.py

```diff
@@ -61,10 +61,7 @@ def _get_dummy_dataset(db_name, number_of_documents, select_fields=[]):
         api_key = (
             "UzdYRktIY0JxNmlvb1NpOFNsenU6VGdTU0s4UjhUR0NsaDdnQTVwUkpKZw"  # read access
         )
-        client = Client(
-            project,
-            api_key,
-        )
+        client = Client(project, api_key, region="old-australia-east")

```

Comment:
Is there a reason...

Reply:
This dataset only exists in the Australian Server